src/SprykerSdk/Zed/AppSdk/Business/Manifest/Builder/ManifestBuilder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Business\Manifest\Builder;
+
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+
+class ManifestBuilder implements ManifestBuilderInterface
+{
+    /**
+     * @var array<string>
+     */
+    protected $transferToManifestTypeMap = [
+        'int' => 'integer',
+    ];
+
+    /**
+     * @param \Generated\Shared\Transfer\ManifestRequestTransfer $manifestRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\ManifestResponseTransfer
+     */
+    public function addManifest(ManifestRequestTransfer $manifestRequestTransfer): ManifestResponseTransfer
+    {
+        $manifestResponseTransfer = new ManifestResponseTransfer();
+
+        $targetFilePath = $manifestRequestTransfer->getManifestPathOrFail();
+        $locale = $manifestRequestTransfer->getManifestOrFail()->getLocaleNameOrFail();
+
+        $targetFile = $targetFilePath . 'en_' . strtoupper($locale) . '.json';
```

Comment:
A locale consists always of a country and a language, you use locale as the language here which doesn't make sense. From CLI you have to pass `en_US` or `de_DE` etc.

Please also add an assertion for a valid local name. Only when the passed locale is of format `[a-z]{2}_[A-Z]{2}` (<- check if this covers all valid locales) you can create the manifest file.

If you don't have a valid locale return a failed response that can be used to print the issue to the console.

Add a test case for this.